enginerpc/engine.proto

```diff
@@ -1,13 +1,52 @@
 syntax = "proto3";
 
 import "google/protobuf/empty.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
 
 service Engine {
   rpc GetVersion(google.protobuf.Empty) returns (Version) {}
+  rpc GetStats(google.protobuf.Empty) returns (Stats) {}
+
+  rpc ProcessServiceCheckResult(ServiceCheck) returns (CommandSuccess) {}
```

Comment:
Perhaps you can add a comment  to update this method further using this :

ProcessCheckResult(Check)

using this message

message Check {
  oneof check {
    HostCheck host =1;
    ServiceCheck svc =2;
  }
}